react/Section/Section.less

```diff
@@ -1,11 +1,74 @@
 @import (reference) "~seek-style-guide/theme";
 
+@gutter-adjustment: 100px;
```

Comment:
This `100px` value feels like a bit of a magic number. Would it make sense to define it using a calculation instead?

Reply:
The definition is 'bigger than the gutter' so I'll go with `@gutter-adjustment: @grid-gutter-width + 50px;` unless there's a better suggestion